app/posts/manage-teacher-training-applications/2021-01-06-closing-an-application.md

```diff
@@ -0,0 +1,151 @@
+
+Providers can close an application by:
+
+- rejecting an application (rejected)
+- not responding to an application before the deadline (rejected)
+- withdrawing an offer (offer withdrawn)
+- marking a condition as not met (conditions not met)
+
+Providers give candidates different types of feedback depending on what they’re doing. For example, when rejecting a candidate, providers fill out structured reasons for rejection. Whereas, when withdrawing an offer, they get a free text box.
+
+It’s not always clear to providers what the difference is between rejection and withdrawal.
+
+And an application can not be always be closed depending on the status. For example, if a candidate stops responding after they accept an offer, there’s no way to close the application besides marking a condition as not met.
+
+## Why are we doing this?
+
+We’re doing this work to:
+
+- make it easier for providers to choose the right action without getting hung up on terminology
+- let providers close an application at any time throughout the application process
+- allow providers to process applications where candidates have disappeared or stopped responding
+- give providers confidence that they won’t get punished about rejecting an application
+- give candidates useful feedback about why they were unsuccessful
+- allow candidates to be withdrawn at their request
+- make sure DfE can accurately differentiate between withdrawals and rejections
+
+## What outcomes are we looking for
+
+- less or no support requests to close applications
+- less or no applications being incorrectly marked as automatically rejected or automatically declined
+- more candidates applying again and being successful when they do
+
+## What we did and why
+
+- Replaced the words ‘reject application’ and ‘withdraw offer’ to ‘Close application’ so that users do not get hung up on terminology
+- Replace ‘Declined’, ‘Rejected’, ‘Offer withdrawn’ and ‘Application withdrawn’ statuses with ‘Closed’
+- Always let users close an application (unless it’s already been closed) so that there’s a consistent way to close an application regardless of the status
+- Kept all questions on the first screen for closing an application as it’s more work to split them up for this round of research
+- We put questions that are not a reflection of the candidate toward the bottom of the form
+- Once an offer has been made, irrelevant questions like safeguarding and performance at interview are removed
+- Let providers give additional feedback when the course is full because if the candidate was poor, then the course being full is not the main reason for closing the application
+- Let users navigate easily to change the status of a condition if they choose ‘Close application’ when in the ‘Accepted’ state
+- Kept the ‘performance at interview’ question in the ‘Received’ state because a provider may not use the interview feature
```

Comment:
I'm not sure what you mean here – how is keeping the question related to whether the provider uses the feature?